Kitodo/src/test/java/org/kitodo/selenium/testframework/pages/TasksPage.java

```diff
@@ -12,9 +12,17 @@
 package org.kitodo.selenium.testframework.pages;
 
 import org.kitodo.selenium.testframework.Pages;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import static org.kitodo.selenium.testframework.Browser.getRowsOfTable;
```

Comment:
`import static` statements should be in front of `import` statements. Please move it.